src/components/CategoryPicker/index.js

```diff
@@ -74,7 +62,8 @@ function CategoryPicker({selectedCategory, policyCategories, policyRecentlyUsedC
             sections={sections}
             selectedOptions={selectedOptions}
             value={searchValue}
-            initialFocusedIndex={initialFocusedIndex}
+            focusedIndex={0}
+            initiallyFocusedOptionKey={lodashGet(_.filter(lodashGet(sections, '[0].data', []), (category) => category.searchText === selectedCategory)[0], 'keyForList')}
```

Comment:
Can we extract the value to separate variable and add a comment explaining the logic for initially focused key, same in TagsPicker